api/types.go

```diff
@@ -161,6 +166,35 @@ type (
 		CallbackURL string   `json:"callbackURL"`
 		Scopes      []string `json:"scopes"`
 	}
+
+	// TPoolResp is the response body for the [GET] /tpool/fee endpoint
+	TPoolResp struct {
```

Comment:
This should be `TPoolResp types.Currency`

Reply:
See comments on other open conversation